libs/libbuiltin/Kconfig

```diff
@@ -10,6 +10,7 @@ config LIB_BUILTIN
 
 config LIB_COMPILER_RT
 	bool
+	depends on ARCH_TOOLCHAIN_CLANG
```

Comment:
but libbuiltin should work with gcc too.

Reply:
This is the error reported after switching to the gcc compiler. Does this error need to be fixed or does it mean that compilation is not supported?
```
AS:  compiler-rt/compiler-rt/lib/builtins/arm/clzdi2.S compiler-rt/compiler-rt/lib/builtins/arm/chkstk.S: Assembler messages:
compiler-rt/compiler-rt/lib/builtins/arm/chkstk.S:28: Error: lo register required -- `sub r12,r12,#4096'
compiler-rt/compiler-rt/lib/builtins/arm/chkstk.S:29: Error: instruction not supported in Thumb16 mode -- `subs r5,r5,#4096'
compiler-rt/compiler-rt/lib/builtins/arm/chkstk.S:30: Error: lo register required -- `ldr r6,[r12]'
make[2]: *** [Makefile:45：bin/compiler-rt/compiler-rt/lib/builtins/arm/chkstk.o] 错误 1
```

Reply:
let's fix it.